src/auth/_aws.js

```diff
@@ -228,3 +231,39 @@ export function formatSignedHeaders(headers) {
 
     return response.slice(0, -1);
 }
+
+
+/**
+ * Returns date in format YYYYMMDD
+ * 
+ * @export
+ * @returns {String}
+ */
+export function getShortDate() {
+    const now = new Date();
+    return `${now.getUTCFullYear()}${pad(now.getUTCMonth() + 1)}${pad(now.getUTCDate())}`;
+}
+
+
+/**
+ * Creates Scope object for signed string i.e. "20130606/us-east-1/s3/aws4_request"
+ * 
+ * @export
+ * @returns {String}
+ */
+export function createScope() {
+    return `${getShortDate()}/${AWS_REGION}/s3/aws4_request`;
+}
+
+
+/**
+ * Performs HmacSHA256 encryption
+ * 
+ * @export
+ * @param {any} message 
+ * @param {any} key 
+ * @returns {String}
+ */
+export function toHmacSHA256(message, key) {
+    return hmacSHA256(message, key);
```

Comment:
see below. I recommend you flip the order of the params sent to the `crypto-js` function here and revert your recent change.